content/docs/deployment/csminstallationwizard/src/index.html

```diff
@@ -61,7 +61,7 @@
                     <select id="array" class="form-select" onchange="validateInput(validateForm, CONSTANTS)" required>
                       <option value="" selected>Select the Array</option>
                       <option value="csi-powerstore">PowerStore</option>
-                      <option value="csi-powermax">PowerMax</option>
+                      <option value="powermax">PowerMax</option>
```

Comment:
Can we use the common "csi-<platform>" name for all drivers or is powermax different for a particular reason?